docs/UserGuide.md

```diff
@@ -114,6 +114,34 @@ WARNING: Please note that `clear` is an irreversible command and all existing da
 
 </div>
 
+### Feature: General Commands
+
+#### Viewing help : `help`
+
+Shows a message explaining how to access the help page.
```

Comment:
Should it be 'shows message explaining how to access the user guide' cos the link is to the UG